_data/hosting_providers.json

```
@@ -2188,5 +2188,15 @@
     "plan": "https://www.easyname.com/en/hosting/free-lets-encrypt-ssl-certificate",
     "reviewed": "2019.7.10",
     "note": ""
+  }
+     {
```

One teeeny tiny nit, then we're good to go!
```suggestion
  {
```